Extensions/xkit_preferences.js

```diff
@@ -407,40 +433,50 @@ XKit.extensions.xkit_preferences = new Object({
 			for (i=0;i<prev_objects.length;i++) {
 
 				if (prev_objects[i].id === id) {
-					return true;
+					if(prev_objects[i].version >= version) {
+						return 1;
+					} else {
+						return 2;
+					}
 				}
 
 			}
 
-			return false;
+			return 0;
 
 		},
 
-		create: function(id, title, message, date, important) {
-
-			if (XKit.extensions.xkit_preferences.news.check(id) === true) {
-				XKit.console.add("News " + id + " could not be pushed: already exists.");
+		create: function(news_object) {
+			if (typeof(news_object.version) === "undefined") { news_object.version = 1; }
```

Comment:
It's honestly just a save-guard if something ever goes wrong. The Builder should already error out if there is no version specified iirc